contributors.json

```diff
@@ -22,6 +22,8 @@
 { "profile_name":"jlstevens", "organisation":"Continuum Analytics"},
 { "profile_name":"jbednar", "organisation":"Continuum Analytics"},
 { "profile_name":"rrenfrow86", "organisation":"Continuum Analytics"},
+{ "profile_name":"Armin Leuprecht", "organisation":"Weneger Center for Climate and Global Change"},
+{ "profile_name":"Tim Michelsen", "organisation":""},
```

Comment:
The extra comma means the about.html page is broken, as malformed JSON is a silent fail in jQuery.